docs/features/custom-ingress-class.md

```diff
@@ -0,0 +1,34 @@
+# Custom Ingress Class
+> **Minimum version:** 1.3.0
+
+Custom ingress class allows you to customize the ingress class selector that AGIC will use when filtering the ingress manifests. AGIC uses `azure/application-gateway` as default ingress class. This  will allow you to target multiple AGICs on a single namespace as each AGIC can now use it's own ingress class.
```

Comment:
Link "Custom ingress class" to https://github.com/Azure/application-gateway-kubernetes-ingress/blob/master/proposals/multiple-gateways-single-cluster.md?